.github/workflows/verify-deploy.yml

```diff
@@ -0,0 +1,77 @@
+name: Verify deployment
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
```

Comment:
This job should be added to the pull-request and push-main orchestration jobs rather than triggering here. It should run last and depend on the e2e-test in both cases.